chainerx_cc/chainerx/python/pymodule.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
```

Comment:
I'm not sure if we need this file.
How about resolving each time directly?

Reply:
Because resolving Python objects each time incurs larger performance overhead.

Reply:
Hmm, I don't think that matters.
By storing them in `static` function-local variables, it should be equivalent to this implementation in terms of peformance.

Reply:
Ah, ok. Do you mean I should store the instance in each caller function instead of declaring in the separate file? It makes sense in terms of performance but I also think that collecting `py::module::import` things in one place is useful for later maintenance. Any idea?

Reply:
We agreed offline to define these functions in this file.

Let's give it more clear name.
How about `py_cached_objects.h`?